src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -100,7 +106,7 @@ internal string GetAssemblyFileVersion()
 
         internal Version GetAssemblyVersion()
```

Comment:
Curious, what is this used for? Perhaps a comment for why this would be used vs. the other one might avoid future confusion.